### PR TITLE
chore: Move connectionstore under pkg/store/connectionstore

### LINKED ISSUE #1045: Move connectionstore under pkg/store/connectionstore

### PR DESCRIPTION
**Pull request by fqutishat**
closes #1045

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: pkg/client/didexchange/client.go [hunk 1]
```diff
@@ -76,7 +76,7 @@ func New(ctx provider) (*Client, error) {
 		return nil, errors.New("cast service to DIDExchange Service failed")
 	}
 
-	connectionStore, err := connectionstore.NewConnectionRecorder(ctx)
+	connectionStore, err := connection.NewConnectionRecorder(ctx)
```
Comment (troyronda) [inline pkg/client/didexchange/client.go]: This probably could have been connection.NewRecorder

File: pkg/client/route/client.go [hunk 2]
```diff
@@ -54,7 +53,7 @@ func New(ctx provider) (*Client, error) {
 		return nil, errors.New("cast service to route service failed")
 	}
 
-	connectionLookup, err := connectionstore.NewConnectionLookup(ctx)
+	connectionLookup, err := connection.NewConnectionLookup(ctx)
```
Comment (troyronda) [inline pkg/client/route/client.go]: This probably could have been connection.NewLookup

Comment (fqutishat) [reply pkg/client/route/client.go]: done

File: pkg/client/didexchange/client.go (hunk 1 above)
Comment (fqutishat) [reply pkg/client/didexchange/client.go]: done

**Pull request merged**